sdks/python/apache_beam/transforms/userstate_test.py

```diff
@@ -415,6 +422,82 @@ def clear_values(self, bag_state=beam.DoFn.StateParam(BAG_STATE)):
         ['extra'],
         StatefulDoFnOnDirectRunnerTest.all_records)
 
+  def test_simple_set_stateful_dofn(self):
+    class SimpleTestSetStatefulDoFn(DoFn):
+      BUFFER_STATE = SetStateSpec('buffer', BytesCoder())
+      EXPIRY_TIMER = TimerSpec('expiry', TimeDomain.WATERMARK)
+
+      def process(self, element, buffer=DoFn.StateParam(BUFFER_STATE),
+                  timer1=DoFn.TimerParam(EXPIRY_TIMER)):
+        unused_key, value = element
+        buffer.add(str(value).encode('latin1'))
```

Comment:
Can we just use StringCoder and avoid the explicit encoding? Or if the input is integers, just store the int itself?